app/main.py

```diff
@@ -1,3 +1,61 @@
-def shop_trip():
-    # write your code here
-    pass
+from app.customer import Customer
+from app.shop import Shop
+
+import json
+
+
+def shop_trip() -> None:
+    with open("app/config.json", "r") as file:
+        data = json.load(file)
+        fuel_price = data["FUEL_PRICE"]
+        cust_info = data["customers"]
+        shop_info = data["shops"]
+
+    customer_list = [
```

Comment:
```suggestion
    customers = [
```
_list suffix is redundant